autopush/tests/test_integration.py

```diff
@@ -183,6 +188,9 @@ def get_notification(self, timeout=0.2):
         except:
             return None
 
+    def get_notification(self, timeout=0.2):
```

Comment:
D'oh, I forgot `__getattribute__` is inherited and you can actually do that. :stuck_out_tongue_winking_eye: